modules/util/source-formatter/src/test/java/com/liferay/source/formatter/processor/UpgradeCatchAllCheckTest.java

```diff
@@ -125,10 +125,14 @@ public void testUpgradeCatchAllCheck() throws Exception {
 			StringUtil.replace(_issueKey, CharPool.DASH, CharPool.UNDERLINE) +
 				".test" + _fileType;
 
-		if (_hasValidUpgradeFiles(_issueKey, _fileType)) {
-			_testUpgradeCatchAllCheck(
-				"upgrade/upgrade-catch-all-check/" + fileName);
-		}
+		Assert.assertTrue(
+			StringBundler.concat(
+				"Missing unit test for Upgrade Automation of issueKey: ",
```

Comment:
> Upgrade Automation

Do we us this language anywhere else? I think it might be clearer to reference the need for a test case for an entry in `replacements.json`, rather than "Upgrade Automation". Thoughts?